vaadin-date-picker-mixin.html

```diff
@@ -10,6 +10,7 @@
 
 <script>
   window.Vaadin = window.Vaadin || {};
+  var isRtl = !!document.querySelector('[dir=rtl]');
```

Comment:
Should this somehow check that it actually affects the date picker element? I mean, in theory, it could be set in a part of the HTML tree which does not contain the date picker. But I suppose that’s very theoretical worry, since usually it’s the whole page that is ltr/rtl.

Reply:
I've changed accordingly to check if the picker is inside a `[dir=rtl]` element.